src/material/core/style/_focus-indicators.scss

```diff
@@ -0,0 +1,57 @@
+// Private sass variables that will be used as reference throughout component stylesheets.
+$private-default-focus-indicator-border-width: 3px;
+$private-default-focus-indicator-border-style: solid;
+$private-default-focus-indicator-border-color: transparent;
+$private-default-focus-indicator-border-radius: 4px;
+
+// Mixin that renders the focus indicator structural styles.
+@mixin private-structural-styling($prefix) {
+  .#{$prefix}-focus-indicator {
+    position: relative;
+
+    &::before {
+      @include layout-common.fill();
+      box-sizing: border-box;
+      pointer-events: none;
+      border: var(
+          --#{$prefix}-focus-indicator-border-width,
+          $private-default-focus-indicator-border-width
+        )
+        var(
+          --#{$prefix}-focus-indicator-border-style,
+          $private-default-focus-indicator-border-style
+        )
+        var(
+          --#{$prefix}-focus-indicator-border-color,
+          $private-default-focus-indicator-border-color
+        );
+      border-radius: var(
+        --#{$prefix}-focus-indicator-border-radius,
+        $private-default-focus-indicator-border-radius
+      );
+    }
+    &:focus::before {
+      content: '';
+    }
+  }
+}
+
+// Mixin that handles null values and defines CSS variables accordingly.
+@mixin _create-variable($name, $value) {
```

Comment:
`_` or `private-`? Maybe a question for @crisbeto

Reply:
Kristiyan recommended `_` since it is only referenced/used in this file (`src/material/core/style/_focus-indicators.scss`)

Reply:
Sounds good!